cli/monitoring/custom-monitoring.yaml

```diff
@@ -28,9 +31,11 @@ create_monitor:
             trailing_window_size: P30D
             trailing_window_offset: P7D
           pre_processing_component: azureml:custom_preprocessor:1.0.0
```

Comment:
This should be the custom metrics component instead of the custom preprocessor, do we still use the pre_processing_component?

Reply:
Any example also for the custom preprocessor? And besides the examples any documentation to tell in which case the customer should follow the custom preprocessor way.